assets/js/blocks/featured-category/index.js

```diff
@@ -146,14 +163,12 @@ registerBlockType( 'woocommerce/featured-category', {
 	 *
 	 * @param {Object} props Props to pass to block.
 	 */
-	edit( props ) {
-		return <Block { ...props } />;
-	},
+	edit: Edit,
 
 	/**
 	 * Block content is rendered in PHP, not via save function.
 	 */
-	save() {
+	save: () => {
```

Comment:
Why was this changed? Was this automatically done by Prettier? I find 13 instances of `save() {` but only this one of `save: () =>`, so perhaps the former is more in line with the repo conventions?